automated/linux/kvm-unit-tests/kvm-unit-tests.yaml

```diff
@@ -24,9 +24,10 @@ metadata:
 params:
     SKIP_INSTALL: "False"
     SMP: "true"
+    GIT_REF: ""
```

Comment:
I think if you pass in "" it won't get defaulted to "master" as intended.

Reply:
Thanks for the comments.
GIT_REF default as master now.
 GIT_REF: "master"